plugins/modules/xenserver_guest.py

```diff
@@ -53,248 +51,249 @@
-    - This parameter is case sensitive.
+      - Name of the VM to work with.
+      - VMs running on XenServer do not necessarily have unique names. The module will fail if multiple VMs with same name are found.
+      - In case of multiple VMs with same name, use O(uuid) to uniquely specify VM to manage.
+      - This parameter is case sensitive.
     type: str
-    aliases: [ name_label ]
+    aliases: [name_label]
   name_desc:
     description:
-    - VM description.
+      - VM description.
     type: str
   uuid:
     description:
-    - UUID of the VM to manage if known. This is XenServer's unique identifier.
-    - It is required if name is not unique.
-    - Please note that a supplied UUID will be ignored on VM creation, as XenServer creates the UUID internally.
+      - UUID of the VM to manage if known. This is XenServer's unique identifier.
+      - It is required if name is not unique.
+      - Please note that a supplied UUID will be ignored on VM creation, as XenServer creates the UUID internally.
     type: str
   template:
     description:
-    - Name of a template, an existing VM (must be shut down) or a snapshot that should be used to create VM.
-    - Templates/VMs/snapshots on XenServer do not necessarily have unique names. The module will fail if multiple templates with same name are found.
-    - In case of multiple templates/VMs/snapshots with same name, use O(template_uuid) to uniquely specify source template.
-    - If VM already exists, this setting will be ignored.
-    - This parameter is case sensitive.
+      - Name of a template, an existing VM (must be shut down) or a snapshot that should be used to create VM.
+      - Templates/VMs/snapshots on XenServer do not necessarily have unique names. The module will fail if multiple templates with same name are
+        found.
+      - In case of multiple templates/VMs/snapshots with same name, use O(template_uuid) to uniquely specify source template.
+      - If VM already exists, this setting will be ignored.
+      - This parameter is case sensitive.
     type: str
-    aliases: [ template_src ]
+    aliases: [template_src]
   template_uuid:
     description:
-    - UUID of a template, an existing VM or a snapshot that should be used to create VM.
-    - It is required if template name is not unique.
+      - UUID of a template, an existing VM or a snapshot that should be used to create VM.
+      - It is required if template name is not unique.
     type: str
   is_template:
     description:
-    - Convert VM to template.
+      - Convert VM to template.
     type: bool
     default: false
   folder:
     description:
-    - Destination folder for VM.
-    - This parameter is case sensitive.
-    - 'Example:'
-    - '  folder: /folder1/folder2'
+      - Destination folder for VM.
+      - This parameter is case sensitive.
+      - 'Example:'
+      - '  folder: /folder1/folder2'
```

Comment:
@bvitnik I think this would be better suited for the EXAMPLES block below, but I am not too familiar with `xenserver` to be sure of where/how to place it.

Reply:
I think it's better to leave it here. It would be hard to spot in the EXAMPLES block. The idea here is to emphasize the format of the value. Not many people know that folders can have arbitrarily deep hierarchy.